portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2771,6 +2778,92 @@ private long _getValidFileEntryTypeId(
 		}
 	}
 
+	private void _inheritRolesPermissions(
+			long propagationParentFolderId, DLFileEntry dlFileEntry)
+		throws PortalException {
+
+		Set<String> commonPermissions = new HashSet<>(
+			Arrays.asList(
+				ActionKeys.DELETE, ActionKeys.PERMISSIONS, ActionKeys.UPDATE,
+				ActionKeys.VIEW));
```

Comment:
I think it should be declared as a constant instead

Reply:
Fix it. Thanks

Reply:
I thinks it is should in DLFileEntryConstants

Reply:
I think these common permissions are belonged to both DLFileEntry and DLFolder, so if we define them in only File Constant, they are not enough meaning. Moreover, this set is used in this class, so it is no need to defined in File Constant.